saleor/graphql/core/scalars.py

```diff
@@ -46,18 +46,29 @@ def parse_value(value):
             return None
 
 
-class PositiveDecimal(Decimal):
+class PositiveDecimal(graphene.Float):
```

Comment:
Removed inheritance because we don't need it, it delegates behavior to Decimal, not super()